ng-dev/perf/workflow/cli.ts

```diff
@@ -34,30 +37,48 @@ function builder(yargs: Argv) {
     .option('name', {
       type: 'string',
       description: 'A specific workflow to run by name',
+    })
+    .option('commit-sha' as 'commitSha', {
+      type: 'string',
+      description: 'The commit sha to associate the measurement with, uploading it to our database',
     });
 }
 
 /** Handles the checkout pull request command. */
-async function handler({configFile, list, name}: WorkflowsParams) {
+async function handler({configFile, list, name, commitSha}: WorkflowsParams) {
   const workflows = await loadWorkflows(join(determineRepoBaseDirFromCwd(), configFile));
 
   if (list) {
     process.stdout.write(JSON.stringify(Object.keys(workflows)));
     return;
   }
 
+  const results: {name: string; value: number}[] = [];
+
   if (name) {
-    const {duration} = await measureWorkflow(workflows[name]);
-    process.stdout.write(JSON.stringify({[name]: duration}));
-    return;
+    const {value} = await measureWorkflow(workflows[name]);
+    results.push({value, name});
+  } else {
+    for (const workflow of Object.values(workflows)) {
+      const {name, value} = await measureWorkflow(workflow);
+      results.push({value, name});
+    }
   }
 
-  const results: {[key: string]: number} = {};
-  for (const workflow of Object.values(workflows)) {
-    const {name, duration} = await measureWorkflow(workflow);
-    results[name] = duration;
+  if (commitSha) {
+    const spinner = new Spinner('Uploading performance results to database');
+    try {
+      for (let {value, name} of results) {
+        await addWorkflowPerformanceResult({
```

Comment:
Can we still print the results?

Reply:
The result is printed on each result inline.  It just is not longer written as a JSON output.  The result of running the command looks like this:

```

 ✓ Rerun a test: 9639.00ms
 ✓ Upload complete

Reply:
Oh nice, didn't see that!